src/uu/echo/src/echo.rs

```diff
@@ -122,34 +122,41 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
         // Final argument must have multiple(true) or the usage string equivalent.
         .setting(clap::AppSettings::TrailingVarArg)
         .setting(clap::AppSettings::AllowLeadingHyphen)
-        .version(VERSION)
-        .usage(SYNTAX)
+        .version(crate_version!())
```

Comment:
we could do that in other different places in coreutils. Would you be interested in doing it? thanks

Reply:
I'll check

Reply:
thanks!